test/jdk/java/lang/Thread/virtual/Starvation.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, 2025 Oracle and/or its affiliates. All rights reserved.
```

Comment:
You missed the comma after 2025 here. This is now causing failures in our CI.

Reply:
Sorry, I create a new issue [JDK-8349876](https://bugs.openjdk.org/browse/JDK-8349876) for that.